pulp_container/app/viewsets.py

```diff
@@ -760,3 +760,37 @@ class ContainerNamespaceViewSet(
             },
         ],
     }
+
+    @extend_schema(
+        description="Trigger an asynchronous delete task",
+        responses={202: AsyncOperationResponseSerializer},
+    )
+    def destroy(self, request, pk, **kwargs):
```

Comment:
we should leave this as a synchronous call. podman does not have a way how to deal with the async tasks, @dkliban PTAL

Reply:
we have also async distribution deletion. we need to brainstorm this more.

Reply:
podman is never going to hit this, as it is part of the pulp api.

Reply:
@mdellweg I will be happy to merge this PR and address the async/sync/locking issue separately. I have filed an issue for it https://pulp.plan.io/issues/8151